src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,74 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.fail;
+
+import core.basesyntax.exception.CustomException;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private static RegistrationService service;
+
+    @BeforeAll
+    static void beforeAll() {
+        service = new RegistrationServiceImpl();
+    }
+
+    @Test
+    void userIsNull_notOk() {
+        User user = null;
+        try {
+            User register = service.register(user);
+        } catch (CustomException e) {
+            System.out.println("Custom exception was catched.");
+            return;
+        }
+        fail();
```

Comment:
Instead of using try-catch with a print statement and fail, use `assertThrows` to directly assert that the expected exception is thrown. This simplifies the test and makes it more readable.